flutter_app/lib/command_constructor.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:dart_mavlink/mavlink.dart';
+import 'package:dart_mavlink/dialects/common.dart';
+
+// ignore_for_file: avoid_print
+
+// Construct command to request a single instance of a message using MAV_CMD_REQUEST_MESSAGE (512)
+MavlinkFrame requestMessage(
+    int sequence, int systemId, int componentId, int messageId, int param2) {
```

Comment:
Add comment explaining what how each parameter is used (for callers of this function).

Reply:
Done.